modules/backlogs/config/locales/en.yml

```diff
@@ -62,7 +62,7 @@ en:
   backlogs:
     add_new_story: "New Story"
     any: "any"
-    backlog_settings: "Backlog Settings"
+    backlog_settings: "Backlogs Settings"
```

Comment:
I'd rather call this "Backlogs settings" (Start "settings" with a lowercase "s")

Reply:
Strictly speaking out of scope: The settings page for backlogs is titled "Definition of done" which is also inconsistent with the name chosen for the menu item.